common/setup.c

```diff
@@ -52,6 +52,7 @@ bool_cmd("debug", opt_debug);
 io_port_t com_ports[2];
 
 const char *kernel_cmdline;
+char cpu_identifier[48];
```

Comment:
nit: magic number; It'd probably make sense to add another byte for explicit `'\0'`